Runtime/Scripts/Sensor.cs

```diff
@@ -14,45 +14,50 @@ public class Sensor : MonoBehaviour
         /// <summary>
         /// Serial of the sensor to handle events from.
         /// </summary>
+        [Tooltip("Serialnumber of the physical sensor.")]
         public string Serial = "";
         /// <summary>
+        /// Radius of the field of view of the sensor. SensMax TAC-B 
+        /// sensors have a 10m radius.
+        /// </summary>
+        [Tooltip("Radius of the visiual indicator of field of view of the sensor. Does not effect the function of this script. SensMax TAC-B sensors have a 10m radius.")]
```

Comment:
Blir dette tooltippet for langt? Kunne kanskje holdt med første delen, og skrive detaljer senere. Skal se hvordan det ser ut i Unity straks.
```suggestion
        [Tooltip("Radius of the visual indicator showing the sensor's field of view. Does not effect the function of this script. SensMax TAC-B sensors have a 10m radius.")]
```